### PR TITLE
Added code documentation + final var cleanup

### PR DESCRIPTION
**Pull request by MarcinOrlowski**
Scope of the PR (each type of change in separate commit)

* Replaced empty string comparison with isEmpty() 
* Added code documentation
* "final var" refactoring

### CODE REVIEW

**Review by davidhutchens — DISMISSED** [review]
Looks good to me.

File: src/main/java/com/cburch/logisim/prefs/PrefMonitorKeyStroke.java [hunk 1]
```diff
@@ -49,6 +69,12 @@ public PrefMonitorKeyStroke(String name, int keycode, int modifier,
     this.canModify = canModify;
   }
 
+  /**
+   * Constructor initializes a preference monitor with multiple keystrokes.
+   *
+   * @param name          The preference's name.
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/prefs/PrefMonitorKeyStroke.java:75]: Fix alignment wrt. next line.

File: src/main/java/com/cburch/logisim/prefs/PrefMonitorKeyStroke.java [hunk 2]
```diff
@@ -165,6 +258,13 @@ public String getDisplayString() {
     return res.toString();
   }
 
+  /**
+   * Handles the preference change events.
+   * Does nothing if newValue is the same as the current value.
+   *
```
Comment (maehne) [inline src/main/java/com/cburch/logisim/prefs/PrefMonitorKeyStroke.java:264]: One empty line as separator should be enough.

**Review by maehne — DISMISSED** [review]
Thanks for the clean-up/documentation effort! I just noticed two minor details.

File: src/main/java/com/cburch/logisim/prefs/PrefMonitorKeyStroke.java (hunk 2 above)
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/logisim/prefs/PrefMonitorKeyStroke.java:264]: Fixed

File: src/main/java/com/cburch/logisim/prefs/PrefMonitorKeyStroke.java (hunk 1 above)
Comment (MarcinOrlowski) [reply src/main/java/com/cburch/logisim/prefs/PrefMonitorKeyStroke.java:75]: Fixed.

**Review by maehne — APPROVED** [review]
Thanks @MarcinOrlowski!

**Pull request merged**